PaddleAudio/examples/dcase2021_task1b/README_LP.md

```diff
@@ -0,0 +1,113 @@
+# Linear-probe classification using CLIP image encoder
```

Comment:
Why separate them into 2 readme?

Reply:
reasons: 
1. Linear-probe is just a small-part of the example, and it is video-only,  thus not the main focus of PaddleAudio
2. The readme file is too long if merged into the readme, and it will distract readers from the main focus: audio-visual
3. The readme file contains code blocks for illustration purpose. The code is not immediately runnable and it relies on third-party repo that is out of our scope.